ci/analyze.sh

```diff
@@ -47,8 +46,7 @@ echo "Using dart from $DART_BIN"
 "$DART" --version
 echo ""
 
-(cd $SKY_ENGINE_DIR && "$DART" pub get --offline)
-"$DART" analyze "$SKY_ENGINE_DIR/lib/ui/ui.dart"
+"$DART" analyze "$FLUTTER_DIR/lib/ui"
```

Comment:
Talked about this offline - there's a slight risk that this will miss things the build rules do e.g. with _embedder.yaml, but it seems to be working fine and it's what we used to do.

If we ever discover we really need to analyze the build output instead of the input, we can copy analysis_options over to the `root_gen_dir` and it should probably work, but that may get broken more easily or confusing.